src/commands/exec.ts

```diff
@@ -9,19 +9,19 @@ export default class Exec extends PercyCommand {
 
   static examples = [
     '$ percy exec -- echo \"percy is running around this echo command\"',
-    '$ percy exec -- bash -c "echo foo && echo bar"'
+    '$ percy exec -- bash -c "echo foo && echo bar"',
```

Comment:
I've gone with requiring a comma on the last items in arrays etc. This matches a decision we made for the Ruby codebase. It allows you to swap lines around easily without having to manage commas.